processors/rename_field_test.go

```diff
@@ -0,0 +1,53 @@
+package processors
+
+import (
+	"testing"
+
+	"github.com/honeycombio/honeycomb-kubernetes-agent/event"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRenameField(t *testing.T) {
+	// ensures FieldRenamer implements the Processor interface
+	var processor Processor
+	processor = &FieldRenamer{}
+
+	err := processor.Init(map[string]interface{}{
+		"original": "time",
+		"new":      "timestamp",
+	})
+	assert.Equal(t, nil, err, "init should not error")
+
+	e := &event.Event{
+		Data: map[string]interface{}{
+			"time": "tomorrow",
+			"msg":  "leave me be",
+		},
```

Comment:
Consider also including a field here named "timestamp" with a value other than "tomorrow," to confirm that we can overwrite an existing field, as opposed to the processor rejecting that collision.

Ideally, that would be a separate test case.